minifi/minifi-bootstrap/src/test/java/org/apache/nifi/minifi/bootstrap/service/GracefulShutdownParameterProviderTest.java

```diff
@@ -46,9 +49,11 @@ class GracefulShutdownParameterProviderTest {
     @NullSource
     @ValueSource(strings = {"notAnInteger", "-1"})
     void testGetBootstrapPropertiesShouldReturnDefaultShutdownPropertyValue(String shutdownProperty) throws IOException {
-        Properties properties = new Properties();
+        BootstrapProperties properties = mock(BootstrapProperties.class);
         if (shutdownProperty != null) {
-            properties.setProperty(GRACEFUL_SHUTDOWN_PROP, shutdownProperty);
```

Comment:
We could eliminate the if with Optional.ofNullable(shutdownProperty).orElse(DEFAULT_GRACEFUL_SHUTDOWN_VALUE)